src/components/home-page/user-gallery-overview/OnsaleItem.vue

```diff
@@ -0,0 +1,741 @@
+      moment(this.auction.expirationDt).isBefore(moment());
+  }
+
+  get coinDetails() {
+    if (!this.auction) {
+      return null;
+    }
+
+    const { minimumBid } = this.auction;
+
+    const coin = auctionCoins.find((coin) => {
+      return coin.tokenAddress.toLowerCase() === minimumBid.tokenPriceAddress;
+    });
+
+    return coin;
+  }
+
+  get endAuctionBtnLabel() {
+    if (!this.auction || !this.coinDetails) {
+      return '';
+    }
+
+    const { highestBid } = this.auction;
+
+    const { label: coin, decimalPlaces } = this.coinDetails;
+
+    const value = blockchainToCurrency(
+      highestBid ? highestBid.amount : 0,
+      decimalPlaces,
+    );
+
+    const amount = this.$n(value, 'decimal', {
+      maximumFractionDigits: decimalPlaces,
+    } as any); // eslint-disable-line @typescript-eslint/no-explicit-any
+
+    return this.$t('dashboard.auctionPage.claimBtn', { amount, coin });
+  }
+
+  get coinDetailsBid() {
+    const coin = auctionCoins.find((coin) => {
+      return coin.tokenAddress.toLowerCase() === this.tokenPriceAddress;
+    });
+
+    if (!coin) {
+      throw new Error('COIN_NOT_FOUND');
+    }
+
+    return coin;
+  }
+
+  get bidValueMinimumBid() {
+    const amount = blockchainToCurrency(
+      this.minimumBidvalue,
+      this.coinDetailsBid.decimalPlaces,
+    );
+
+    return this.$n(amount, 'decimal', {
+      maximumFractionDigits: this.coinDetailsBid.decimalPlaces,
+    } as any); // eslint-disable-line @typescript-eslint/no-explicit-any
+  }
+
+  get bidValueHighestBid() {
+    const amount = blockchainToCurrency(
+      this.highestBidvalue,
+      this.coinDetailsBid.decimalPlaces,
+    );
+
+    return this.$n(amount, 'decimal', {
+      maximumFractionDigits: this.coinDetailsBid.decimalPlaces,
+    } as any); // eslint-disable-line @typescript-eslint/no-explicit-any
+  }
+
+  async loadAuctionData() {
+    this.loadingGoToAuctionId = true;
+
+    const response = await api
+      .get<IAuctionItem[]>(`auctions?item._id=${this.art._id}`);
+
+    const auction = response.data.pop();
+    if (!auction) {
+      return;
+    }
+
+    this.auction = auction;
+    this.goToAuctionId = this.auction._id;
+    this.index = this.auction.index;
+    this.auctionBidbackRate = await this.bidBackPirsSystem.getBidbackRate(this.index);
+    this.imagePirsRate = await this.bidBackPirsSystem.getInvestorPirsRate(this.index);
+    this.loadingGoToAuctionId = false;
+    this.tokenPriceAddress = this.auction.minimumBid.tokenPriceAddress;
+    this.minimumBidvalue = this.auction.minimumBid.amount;
+    this.lastValueBid = this.auction.bids.length;
+    this.tokenSymbol = this.auction.minimumBid.tokenSymbol;
+    this.highestBidvalue = this.auction.highestBid.amount;
+  }
+
+  get now() {
+    return now.value;
+  }
+
+  loadData() {
+    this.wasLiked =
+      (this.art.likers as string[]).filter(
+        (liker) => liker.toLowerCase() === this.account.toLowerCase(),
+      ).length !== 0;
```

Comment:
```suggestion
    this.wasLiked = this.art.likers.filter((liker) => {
      return liker.toLowerCase() === this.account.toLowerCase();
    }).length !== 0;
```
Isso vai causar um erro no TS, pois o tipo da propriedade `likers` está errado, você pode corrigir o tipo de `likers`.